app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -18,6 +18,7 @@ def global_registration_vars
     location = Location.authorized(:view_locations).find(params['location_id']) if params['location_id'].present?
     host_group = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']) if params["hostgroup_id"].present?
     operatingsystem = Operatingsystem.authorized(:view_operatingsystems).find(params['operatingsystem_id']) if params["operatingsystem_id"].present?
+    ssl_ca_file = File.read(Setting[:ssl_ca_file]) rescue nil
```

Comment:
could we rename the variable to `ssl_ca_cert` please? this is the content of the file